style.css

```diff
@@ -1,19 +1,103 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@500&display=swap');
+
+/* global setting for body contents */
 * {
   margin: 0;
   padding: 0;
+  text-decoration: none;
+  list-style: none;
+  font-family: 'Poppins', sans-serif;
 }
 
-/* Set the  size for the home page */
 body {
-  max-width: auto;
-  height: 100vh;
+  background-color: #f5f5f5;
+}
+
+/* Design and set the main page to center */
+main {
   background-color: white;
+  max-width: 375px;
+  height: 718px;
+  margin: auto;
+}
+
+/* set position */
+header {
+  position: sticky;
+  top: 0;
+}
+
+/* position mobile icons */
+.mobile-icons {
+  display: flex;
+  justify-content: space-between;
+}
+
+/* style mobile icons */
+.mobile-icons p,
+.time {
+  margin: 16px 0 0 21px;
+  font-weight: lighter;
+}
+
+/* position navigation container */
+nav {
+  display: flex;
+  justify-content: space-between;
+}
+
+/* styling for logo name  */
+nav .logo {
+  color: #6070ff;
+  font-size: 18px;
+  font-weight: bolder;
+  margin: 30px 0 0 24px;
 }
 
-/* Style the header text  */
-h1 {
+nav .menu-button {
+  margin: 30px 36px 0 0;
+}
+
+.headline {
+  background-image: url("images/Header_bg.svg");
+  position: absolute;
+  top: 16%;
+}
```

Comment:
- [ ] Kindly make sure the background image is not scrollable, it should be fixed on the page, It's better to use the background-image property for placing the image 🤗 
- _If an image is not part of the content (aesthetic reasons) use a CSS background image. [Check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info._

- [ ] KIndly make sure your background image looks like the same with the original Figma design 🎨 

- Your design:

![18](https://user-images.githubusercontent.com/79058364/185665073-e5fa8b27-d569-4bcc-a965-16d2f3f39f78.png)

- Figma design:

![image](https://user-images.githubusercontent.com/79058364/185665376-d0659000-d7cf-400d-b356-0e92f87aa870.png)